.github/workflows/python-ci-wheel.yml

```diff
@@ -92,7 +92,6 @@ jobs:
           CIBW_BUILD: ${{ env.CIBW_BUILD_IDENTIFIER }}
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
           CIBW_MANYLINUX_I686_IMAGE: manylinux2014
-          CIBW_BEFORE_ALL_LINUX: yum install swig -y
```

Comment:
have you tested this? We spent quite some time to get this action running, so I want to make sure it still builds before I merge.

Reply:
This was based on a comment by @musicEnfanthen but he tested it without success. I'll get back to this later.